WordPress/gutenberg
# Command Palette: fix incorrect path and snackbar message when template part is deleted

Fixes: #52023

## What?
This PR moves you to the Template Parts List page when you delete a template part via Command Palette.

## Why?
When you delete a template part from the Command Palette, the destination `path` is `wp_template_part`. However, with the introduction of Library page in the latest Gutenberg, the following page do not exist.

![wp_template_part](https://github.com/WordPress/gutenberg/assets/54422211/e92fed5e-e7fe-4b39-99fa-d600a8e383c8)

## How?
I have changed the path so that when a template part is deleted, it goes to the Template Parts List page (`wp_template_part/all`).
Another idea might be the Library page.

## Testing Instructions

- Create a template part in Library menu.
- Open the Command Palette.
- Click "Delete template part".
- Confirm that you are taken to the Template Parts List page.

**Note:** The snackbar shows `"undefined" deleted`, but this issue has already been reported on #51969.

## Screenshots or screencast 

### trunk

https://github.com/WordPress/gutenberg/assets/54422211/a1687eef-cb07-465e-8dac-24e4f871cf01

### This PR

https://github.com/WordPress/gutenberg/assets/54422211/79a665ce-bf15-4147-88fe-9578ec9808d9
